recipe/meta.yaml

```diff
@@ -7,11 +7,11 @@ package:
 source:
   fn: glue-vispy-viewers-{{version}}.tar.gz
   url: https://pypi.io/packages/source/g/glue-vispy-viewers/glue-vispy-viewers-{{version}}.tar.gz
-  sha256: 3bc00472624e53c2d5ecb50d7bce29acdb2e6f6459694188044daa62c76ae8cf
+  sha256: 9d126735b1c1b08a56e02e11d0eba20c040ff934c8d1fd6644e7ee085a420569
 
 build:
-  noarch: python
   number: 0
+  skip: True  # [py<37 or ppc64le or s390x]
```

Comment:
This package is compatible with python 3.6: https://github.com/glue-viz/glue-vispy-viewers/blob/v1.0.5/setup.cfg#L23

Also, can you leave a comment about why we are not building for `ppc64le` and `s390x`?